graphitron-codegen-parent/graphitron-java-codegen/src/main/java/no/sikt/graphql/schema/ProcessedSchema.java

```diff
@@ -622,7 +622,7 @@ public boolean hasRecord(GenerationField field) {
      * @return Is this an ordered multi-key query?
      */
     public boolean isOrderedMultiKeyQuery(GenerationField field) {
-        return field.isIterableWrapped() && field.isResolver() && hasJavaRecord(field.getContainerTypeName());
+        return field.isIterableWrapped() && field.isResolver() && (hasJavaRecord(field.getContainerTypeName()) || !getRecordType(field.getContainerTypeName()).hasTable());
```

Comment:
```suggestion
        return field.isIterableWrapped() && field.isResolver() && (hasJavaRecord(field.getContainerTypeName()) || !isObjectWithPreviousTableObject(field.getContainerTypeName()));
```
Jeg tror dette blir mer riktig + vi får ikke NullPointerException hvis container typen ikke skulle være en record

Jeg ser at noen tester feiler med denne endringen, men det tror jeg bare er fordi de ikke har rotspørring. Så på de feilende testene kan du bare legge til en rotspørring med `@notGenerated`, så de blir med i `objectWithPreviousTable`-mappet